docs/core/install/includes/linux-install-50-apt.md

```diff
@@ -0,0 +1,34 @@
+
+### Install the SDK
+
+.NET SDK allows you to develop apps with .NET. If you install .NET SDK, you don't need to install the corresponding runtime. To install .NET SDK, run the following commands:
```

Comment:
Suggestion to be applied throughout the PR. 

```suggestion
.NET SDK allows you to develop apps with .NET. If you install the .NET SDK, you don't need to install the corresponding runtime. To install the .NET SDK, run the following commands:
```

Reply:
Thanks for all the review comments! I believe it's just .NET SDK/Runtime because it's the product name?

Reply:
My expectation is also to keep `the`. (Same as referring to the Runtime later in the file.) I think I expect it because `.NET [Core]` is the abstract product but `the .NET [Core] SDK` is a concrete thing you can install.

Either way, I think we should be consistent between the .NET Runtime and the .NET SDK.

(I also capitalize Runtime when I talk about it because it's the concrete product name for that side of things, but I don't necessarily object to lowercase. It just might be obscuring some grammar here (or at least how I see the grammar). 😄)

Reply:
I would also start this sentence with the:

```suggestion
The .NET SDK allows you to develop apps with .NET. If you install the .NET SDK, you don't need to install the corresponding runtime. To install the .NET SDK, run the following commands:
```

Reply:
I know I had discussions in the past with @gewarren and others about the capitalization of Runtime, and we settled on capitalizing.

I remember also discussing adding or not the in front of things. We dropped the from in front of .NET Framework because that's a product name, but I'd consider SDK and Runtime a complement to that (even though still a product name). I don't know, it just reads weird to me without it. @tdykstra usually has some good points when it comes to these things.

Reply:
I agree that ".NET SDK" or ".NET runtime" sounds strange without "the" in front. Capitalizing Runtime seems to go against our usual capitalization rules though.

Notice sentences like this one that occurs a little farther down:  

>ASP.NET Core Runtime allows you to run apps that were made with .NET and .NET Core that didn't provide the runtime.

The end of the sentence "the runtime" is out of sync with "ASP.NET Core Runtime" at the start, and it's the end of the sentence that seems to better fit standard rules of grammar and capitalization.

Reply:
Interesting. I think I would feel the same about "To install *.NET Framework*, [...]" and "To install *the .NET Framework SDK*, [...]". But there is no such thing (AFAIK). I don't know if this is a valid reason to choose one way or another, but it seems like the general pattern I'm expecting.